upstream/CVE-2020-8557.json

```
@@ -0,0 +1,56 @@
+{
+  "id": "CVE-2020-8557",
+  "modified": "2020-07-13T18:39:08Z",
+  "published": "2020-07-13T18:39:08Z",
+  "summary": "Node disk DOS by writing to container /etc/hosts",
+  "details": "The Kubernetes kubelet component in versions 1.1-1.16.12, 1.17.0-1.17.8 and 1.18.0-1.18.5 do not account for disk usage by a pod which writes to its own /etc/hosts file. The /etc/hosts file mounted in a pod by kubelet is not included by the kubelet eviction manager when calculating ephemeral storage usage by a pod. If a pod writes a large amount of data to the /etc/hosts file, it could fill the storage space of the node and cause the node to fail.",
```

We talked with @knqyf263 about `details` and `references` fields - https://github.com/aquasecurity/vuln-list-k8s/pull/62#discussion_r1343516124

```suggestion
  "details": "The Kubernetes kubelet component in versions 0-1.16.12, 1.17.0-1.17.8 and 1.18.0-1.18.5 do not account for disk usage by a pod which writes to its own /etc/hosts file. The /etc/hosts file mounted in a pod by kubelet is not included by the kubelet eviction manager when calculating ephemeral storage usage by a pod. If a pod writes a large amount of data to the /etc/hosts file, it could fill the storage space of the node and cause the node to fail.",
```